be/test/olap/rowset/segment_v2/segment_test.cpp

```diff
@@ -606,6 +606,173 @@ TEST_F(SegmentReaderWriterTest, TestDefaultValueColumn) {
     }
 }
 
+void set_column_value_by_type(FieldType fieldType, int src, char* target, size_t _length = 0) {
+    if (fieldType == OLAP_FIELD_TYPE_CHAR) {
+        char* src_value = &std::to_string(src)[0];
+        int src_len = strlen(src_value);
+
+        auto* dest_slice = (Slice*)target;
+        dest_slice->size = _length;
+        dest_slice->data = new char[dest_slice->size];
```

Comment:
You could use an `Arena` to allocate string data, otherwise you got memory leak

Reply:
👌